packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -74,8 +85,38 @@ class Braze {
     return window.brazeQueue === null;
   }
 
+  setUserAlias() {
```

Comment:
there are 3 log statements here but only 1 of them works as first invocation of log message will set hasLoggedErrorForAlias to true

Reply:
Yeah at a time one these three scenarios can happen - so will log for that specific scenario